website/mails/mails.py

```diff
@@ -232,3 +285,35 @@ def send_mail(to_addr, mail, mimetype='plain', from_addr=None, mailer=None,
     'welcome',
     subject='Welcome to the Open Science Framework'
 )
+
+# Predefined emails with callbacks
+NO_ADDON = {
```

Comment:
I don't believe so. They represent a email template, much like the other email templates in the same file, except with a callback attached to it. This was also done to so that ```queue_mail()``` would match the syntax of ```send_mail()```